packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_typography.scss

```diff
@@ -1,79 +1,83 @@
+@use "sass:map";
 //
 // DISCLAIMER:
 // Do not change this file because it is core styling.
 // Customizing core files will make updating Atlas much more difficult in the future.
 // To customize any core styling, copy the part you want to customize to styles/web/sass/app/ so the core styling is overwritten.
 //
 
+@use "../../exclusions-main" as exclusions;
 @import "../../variables";
```

Comment:
We need to convert this to an `@use` statement where we import the merged variables, like we do in `exclusions-main`. We can then replace all `@import` statements from Atlas. 🎉